### PR TITLE
Popover mousemove event throttled

### PR DESCRIPTION
**Pull request by LucasIcarus**
︿(￣︶￣)︿

### CODE REVIEW

File: packages/zent-popover/src/trigger/HoverTrigger.js [hunk 1]
```diff
@@ -161,7 +162,7 @@ function makeHoverLeaveRecognizer({ leaveDelay, onLeave, isOutSide }) {
             state.transit(HoverState.Started);
           }
         }
-      }
+      }, 149)
```
Comment (cpylua) [inline packages/zent-popover/src/trigger/HoverTrigger.js]: Why 149?

Comment (LucasIcarus) [inline packages/zent-popover/src/trigger/HoverTrigger.js]: default show/hide delay is 150..

set it to 149 just for test... lol

Comment (LucasIcarus) [inline packages/zent-popover/src/trigger/HoverTrigger.js]: default show/hide delay is 150..

set it to 149 just for test... lol

Comment (cpylua) [inline packages/zent-popover/src/trigger/HoverTrigger.js]: 跟延迟没关系的。一般16就好了

**Comment by LucasIcarus** [comment]
已经改了。

**Pull request merged**